classes/pdf/HTMLTemplateSupplyOrderForm.php

```diff
@@ -73,8 +73,6 @@ public function __construct(SupplyOrder $supply_order, Smarty $smarty)
         $this->date = Tools::displayDate($supply_order->date_add);
 
         $this->title = Context::getContext()->getTranslator()->trans('Supply order form', [], 'Shop.Pdf');
-
-        $this->shop = new Shop((int) $this->order->id_shop);
```

Comment:
I don't think if we should have it removed or add a property. It might be good to have it available.

Reply:
I think we must add it when we need it.